contracts/StrategyCurve3crv.sol

```diff
@@ -40,19 +63,13 @@ abstract contract StrategyCurveBase is BaseStrategy {
 
     /* ========== STATE VARIABLES ========== */
     // these should stay the same across different wants.
+    // TODO: in theory no longer required, but need confirmation
```

Comment:
Because we don't swap these in the contract anymore, we don't need to instantiate these tokens. Please confirm